pygeo/mphys/mphys_dvgeo.py

```diff
@@ -84,23 +130,29 @@ def compute(self, inputs, outputs):
         # next time the jacvec product routine is called
         self.update_jac = True
 
-    def nom_addChild(self, ffd_file):
+    def nom_addChild(self, ffd_file, DVGeoName=None):
+        # if we have multiple DVGeos use the one specified by name
+        if self.multDVGeo:
+            DVGeo = self.DVGeos[DVGeoName]
+        else:
+            DVGeo = self.DVGeo
+
         # can only add a child to a FFD DVGeo
-        if self.geo_type != "ffd":
+        if DVGeo.geoType != "ffd":
             raise RuntimeError(
-                f"Only FFD-based DVGeo objects can have children added to them, not type:{self.geo_type}"
+                f"Only FFD-based DVGeo objects can have children added to them, not type:{DVGeo.geoType}"
```

Comment:
Similarly, `DVGeo.geoType` here can be replaced with `type(DVGeo)` or `type(DVGeo).__name__`, which will be just the class name.